Examples/Examples/FormExampleView.swift

```diff
@@ -29,17 +29,20 @@ struct FormExampleView: View {
     
     /// The form view model provides a channel of communication between the form view and its host.
     @StateObject private var formViewModel = FormViewModel()
-    
+    @State private var featureForm: FeatureForm?
+
```

Comment:
```suggestion
    
    /// The form being edited in the form view.
    @State private var featureForm: FeatureForm?
    
```

Or some similar doc comment.